docs/checkit.js

```diff
@@ -91,6 +88,80 @@ function CheckIt() {
 
 };
 
+CheckIt.prototype.addMonth = function() {
+    // Add a single month to the calendar
+    
+    // Get the active Calendar information because you need to know what the 
+    // last month is as well as the endDate.
+    this.store.getActive()
+        .then(function (activeCalendarId) {
+            this.store.loadById(activeCalendarId)
+                .then(function (activeCalendarState) {
+                   if (activeCalendarState !==  null) {
+                       var state = activeCalendarState;
+                       console.log(state);
+                       // Create a calendar so that you can manipulate this calendar State
+                       var calendar = new Calendar(state);
+                       // Add a month to this calendar, function has the same name
+                       // as the one we are currently in. Bad practice?
```

Comment:
It's not bad practice if it clearly describes what you are trying to do. I think it's probably fine. If you don't like it, perhaps you could expand the names to be more precise?